tests/pykafka/test_simpleconsumer.py

```diff
@@ -44,7 +44,8 @@ def _get_simple_consumer(self, **kwargs):
         consumer.stop()
 
     def test_consume(self):
-        with self._get_simple_consumer() as consumer:
+        with self._get_simple_consumer(auto_offset_reset=OffsetType.EARLIEST,
+                                       reset_offset_on_start=True) as consumer:
```

Comment:
Was this test broken by these changes? Mostly curious, but also it makes me feel like I don't fully understand the changes.